errata/views.py

```diff
@@ -7,7 +7,7 @@
 from rest_framework.filters import OrderingFilter
 from django_filters.rest_framework import DjangoFilterBackend, FilterSet
 from django.shortcuts import render, redirect
-from .models import Errata
+from .models import Errata, NEW
```

Comment:
@edwoodward Are you using this `NEW` field anywhere?

Reply:
If you do need this, I would recommend importing `ERRATA_STATUS` and using `ERRATA_STATUS.NEW`

Reply:
@mwharrison Thought I would need it until I figured out how the filters work. It is removed.